examples/eight_schools/main_mcmc.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+import torch
+
+import data
+import pyro
+import pyro.distributions as dist
+import pyro.poutine as poutine
+from pyro.infer import EmpiricalMarginal
+from pyro.infer.mcmc import MCMC, NUTS
+
```

Comment:
Please add the following lines so as to enable validation and fix the RNG seed for deterministic results.

```
pyro.enable_validation(True)
pyro.set_rng_seed(0)
```